src/tr.i18n.js

```diff
@@ -306,38 +303,38 @@ module.exports = {
     'Engine': 'Motor',
     'Black': 'Siyah',
     'Change Player': 'Oyuncuyu Değiştir',
-    'White to play': 'Beyaz Oynar',
-    'Black to play': 'Siyah Oynar',
+    'White to play': 'Sıra Beyazda',
+    'Black to play': 'Sıra Siyahta',
     'White': 'Beyaz'
   },
   'ScoringBar': {
-    'Details': 'Detay',
+    'Details': 'Ayrıntılar',
     'Draw': 'Berabere',
     'Please select dead stones.': 'Lütfen ölü taşları seçin.',
-    'Toggle group status.': 'Grup statüsü geçişi.'
+    'Toggle group status.': 'Grup durumunu göster/gizle.'
```

Comment:
After spending more time by using program i have realized that "Grup durumunu değiştir" is more proper translation.